README.md

```diff
@@ -36,4 +42,5 @@ the checker into your own project:
 
 [1]: https://security.symfony.com/
 [2]: https://github.com/FriendsOfPHP/security-advisories
-[3]: https://get.sensiolabs.org/security-checker.phar
+[4]: https://get.sensiolabs.org/security-checker.phar
+[3]: https://symfony.com/download
```

Comment:
Does the order of the list number matter?

Reply:
no, but I've reordered them :)